frontend/public/components/utils/_copy-to-clipboard.scss

```diff
@@ -13,6 +13,7 @@
 
 .co-copy-to-clipboard__text {
   padding-right: 50px;
+  line-height: 1.4;
```

Comment:
YES!  

Nit:  move new declaration to line 15 (alphabetize) and use `$line-height-base` instead of `1.4`.